### PR TITLE
added the syndesis version properties file as preparation for further…

### PR DESCRIPTION
**Pull request by lhein**
… syndesis work

Signed-off-by: Lars Heinemann <lhein.smx@gmail.com>

# Pull Request Checklist

After this checklist is all checked or PR provides explanations for possible pass-through, please put the label "Ready for review" on the PR.

## General

- [ ] Did you use the Jira Issue number in the commit comments?
- [x] Did you set meaningful commit comments on each commit?
- [x] Did you sign off all commits for this PR? (git commit -s -m "jira issue number - your commit comment")

## Functional

- [ ] Did the CI job report a successful build?
- [ ] Is the non-happy path working, too?

## Maintainability

- [ ] Are all Sonar reported issues fixed or can they be ignored?
- [ ] Is there no duplicated code?
- [ ] Are method-/class-/variable-names meaningful?

## Tests

- [ ] Are there unit-tests?
- [ ] Are there integration tests (or at least a jira to tackle these)?
- [ ] Do we need a new UI test?

## Legal

- [ ] Have you used the correct file header copyright comment?
- [ ] Have you used the correct year in the headers of new files?

### CODE REVIEW

**Comment by lhein** [comment]
base for all syndesis PRs

File: configuration/igniteVersionToDisplayName.properties [hunk 1]
```diff
@@ -0,0 +1,2 @@
+1.2.9.fuse-000001-redhat-1=1.2.9.fuse-000001-redhat-1 (Fuse Ignite TP3)
```
Comment (apupier) [inline configuration/igniteVersionToDisplayName.properties]: isn't Fuse Online finally the chosen name? https://www.redhat.com/en/explore/fuse-online

**Comment by apupier** [comment]
build failed with

```
!ENTRY org.eclipse.jst.j2ee 4 0 2018-03-06 16:21:01.961
!MESSAGE 
!STACK 0
java.lang.ArrayIndexOutOfBoundsException
	at org.eclipse.emf.common.util.BasicEList.toArray(BasicEList.java:255)
	at java.util.ArrayList.<init>(ArrayList.java:177)
	at org.eclipse.wst.common.internal.emf.utilities.ResourceIsLoadingAdapter.findAdapter(ResourceIsLoadingAdapter.java:47)
	at org.eclipse.wst.common.internal.emf.resource.CompatibilityXMIResourceImpl.waitForResourceToLoadIfNecessary(CompatibilityXMIResourceImpl.java:381)
	at org.eclipse.wst.common.internal.emf.resource.CompatibilityXMIResourceImpl.isLoaded(CompatibilityXMIResourceImpl.java:373)
	at org.eclipse.wst.common.componentcore.internal.ModuleStructuralModel.getAndLoadLocalResource(ModuleStructuralModel.java:375)
	at org.eclipse.wst.common.internal.emfworkbench.integration.EditModel.getResource(EditModel.java:683)
	at org.eclipse.wst.common.componentcore.internal.ModuleStructuralModel.getPrimaryResource(ModuleStructuralModel.java:332)
	at org.eclipse.wst.common.componentcore.internal.ModuleStructuralModel.prepareProjectModulesIfNecessary(ModuleStructuralModel.java:240)
	at org.eclipse.wst.common.componentcore.internal.ModuleStructuralModel.getPrimaryRootObject(ModuleStructuralModel.java:119)
	at org.eclipse.wst.common.componentcore.internal.StructureEdit.getComponentModelRoot(StructureEdit.java:471)
	at org.eclipse.wst.common.componentcore.internal.StructureEdit.getWorkbenchModules(StructureEdit.java:506)
	at org.eclipse.wst.common.componentcore.internal.StructureEdit.getComponent(StructureEdit.java:949)
	at org.eclipse.jst.j2ee.internal.common.classpath.J2EEComponentClasspathUpdater$FindNodeJob.findNode(J2EEComponentClasspathUpdater.java:234)
	at org.eclipse.jst.j2ee.internal.common.classpath.J2EEComponentClasspathUpdater$FindNodeJob.findNode(J2EEComponentClasspathUpdater.java:240)
	at org.eclipse.jst.j2ee.internal.common.classpath.J2EEComponentClasspathUpdater$FindNodeJob.findNode(J2EEComponentClasspathUpdater.java:240)
	at org.eclipse.jst.j2ee.internal.common.classpath.J2EEComponentClasspathUpdater$FindNodeJob.access$0(J2EEComponentClasspathUpdater.java:226)
	at org.eclipse.jst.j2ee.internal.common.classpath.J2EEComponentClasspathUpdater$FindNodeJob$1.run(J2EEComponentClasspathUpdater.java:265)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:42)
	at org.eclipse.jst.j2ee.internal.common.classpath.J2EEComponentClasspathUpdater$FindNodeJob.run(J2EEComponentClasspathUpdater.java:258)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:56)
```

it is due to known issue: https://issues.jboss.org/browse/FUSETOOLS-2833

File: configuration/igniteVersionToDisplayName.properties (hunk 1 above)
Comment (apupier) [reply configuration/igniteVersionToDisplayName.properties]: I created https://issues.jboss.org/browse/FUSETOOLS-2854 as discussed on chat to let change name if needed after the work on the wizard refactoring

Comment (lhein) [reply configuration/igniteVersionToDisplayName.properties]: thanks

**Comment by lhein** [comment]
testPR

**Comment by apupier** [comment]
let's go merging this PR as it is only adding a new file which is not referenced anywhere yet to unblock other depending PRs
the failing build was due to know unrelated issue and after to a CI slave failure.

**Pull request merged**